apps/fz_http/lib/fz_http/devices/device.ex

```diff
@@ -225,6 +247,14 @@ defmodule FzHttp.Devices.Device do
 
   defp validate_in_network(changeset, :ipv6), do: changeset
 
+  defp netmask_error(changeset, ip_type) do
+    add_error(
+      changeset,
+      ip_type,
+      "Only IPs without netmask are supported, try using the Allowed IPs field for routing within the network"
```

Comment:
I think we want to leave out the AllowedIPs note since we don’t support that just yet.